openmetadata-ui/src/main/resources/ui/cypress/common/Utils/Annoucement.ts

```diff
@@ -75,6 +75,10 @@ export const createAnnouncement = (announcement) => {
     announcement.title
   );
   cy.goToHomePage();
+
+  cy.get('[data-testid="announcement-container"]')
```

Comment:
Can we assert on URL to redirect it correclty here?

Reply:
Added a check after redirect, will cover this portion in playwrite!